uber/server.py

```diff
@@ -60,11 +60,7 @@ def common_js(self):
 
     static_views = StaticViews()
 
-_sections = [path.split('/')[-1][:-3] for path in glob(os.path.join(c.MODULE_ROOT, 'site_sections', '*.py'))
-                                      if not path.endswith('__init__.py')]
-for _section in _sections:
-    _module = importlib.import_module('uber.site_sections.' + _section)
-    setattr(Root, _section, _module.Root())
+mount_site_sections(c.MODULE_ROOT)
```

Comment:
Just to confirm, does this make it possible to insert new classes/functions into existing site sections? It doesn't really look like it does.